app/controllers/admin/users_controller.rb

```diff
@@ -1,27 +1,69 @@
+# frozen_string_literal: true
+
 class Admin::UsersController < Admin::AdminController
   active_scaffold :user do |config|
-    config.actions.exclude :create, :delete
-    config.columns = %I[email first_name last_name
-                        memberships default_affiliate created_at
-                        updated_at approval_status]
-    config.update.columns = %I[ email first_name last_name organization_name
-                                is_affiliate_admin is_affiliate approval_status
-                                default_affiliate welcome_email_sent]
-    config.list.sorting = { :created_at => :desc }
-    config.columns[:is_affiliate_admin].description = "Set this to true to make the user an Administrator, and give them access to the Admin Center."
-    config.columns[:is_affiliate].description = "Set this to true to make the user an Affiliate, and give them access to the Affiliate Center."
+    config.actions.exclude :create, :delete, :search
+    config.actions.add :export, :field_search
+
+    config.columns = %I[
+      email
+      first_name
+      last_name
+      affiliate_names
+      memberships
+      default_affiliate
+      created_at
+      updated_at
+      approval_status
+    ]
```

Comment:
There's no need to set the parent `config.columns` array. We just need to add the virtual column so that it's available for the other actions:
```ruby
config.columns << :affiliate_names
```

Reply:
Changed.